server/webhook_test.go

```diff
@@ -0,0 +1,88 @@
+package server
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendToWebhookIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	s := &Server{
+		StartTime: time.Now(),
+	}
+
+	validPayload := &Payload{Username: "mattermod", Text: "test"}
+	expectedMattermostPayload := "ok"
```

Comment:
I don't think there's a need to do this. But if you want it, I'll suggest to just use a byte slice. That'll prevent the double-allocation to strings.